web-client/src/presenter/sequences/JudgeActivityReport/getCavAndSubmittedCasesForJudgesSequence.ts

```diff
@@ -0,0 +1,5 @@
+import { getSubmittedAndCavCasesByJudgeAction } from '../../actions/JudgeActivityReport/getSubmittedAndCavCasesByJudgeAction';
```

Comment:
combine sequences, paralle actions

Reply:
Hey @rachelschneiderman 
 I just remembered why the sequences are separated.. If I add `getSubmittedAndCavCasesByJudgeAction` to `submitJudgeActivityReportSequence` then every time a page request is made (for cav and submitted cases), the calls to make the other reports will be made, which isn't necessary. Please let me know if that clarifies your question.

Reply:
Resolved. I have added the `getSubmittedAndCavCasesByJudgeAction` to `submitJudgeActivityReportSequence`.